packages/shared-ux/markdown/impl/markdown.test.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { Markdown } from './markdown';
+import { render } from 'enzyme';
+
+describe('shared ux markdown component', () => {
+  it('renders for editor', () => {
+    const component = render(<Markdown readOnly={false} />);
+    expect(component).toMatchSnapshot();
+  });
+
+  it('renders for displaying a readonly message', () => {
+    const component = render(<Markdown readOnly markdownContent="error message" />);
```

Comment:
nit: I would not use "error" in this test content as it is not relevant for the test (we are not testing an error state). `"Sample message"` would be enough.

Can we also test that the message is rendered instead of relying on `toMatchSnapshot()` ?

Something like:

```
expect(component.text()).toContain('Sample message');
```

Reply:
Good call thanks!